mysql-to-kv/ingest_from_mysql.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+import pymysql
+import requests
+import json
+import pandas as pd
+from requests.auth import HTTPBasicAuth
+
+
+def _pack_putitem_body(key, row):
+    """
+    PutItem request body formater
+    """
+    dict_item = dict()
+    for k, v in row.iteritems():
+        dict_item[k] = {'N' if isinstance(v, int) else 'S': '{}'.format(v)}
+    body = {
+            "Key": {
+                "ID": {
+                    "N": '{}'.format(key)
+                }
+            },
+            "Item": dict_item
+        }
+    return body
+
+
+def _putitem(url, body, user, password):
+    """
+    PutItem request
+    """
+    headers = {'Content-Type': 'application/json', 'X-v3io-function': 'PutItem'}
+    res = requests.post(url=url, data=json.dumps(body), headers=headers, auth=HTTPBasicAuth(user, password))
+    return res
+
+
+def handler(context, event):
```

Comment:
1. handle should be first function (convention)
2. you're creating a connection per invocation, use init_context (should be second function after handler - convention):

```
def init_context(context):
	
	# connect to database
	connection = ...

	# set connection as member of context
	setattr(context, 'connection', connection)
```

Then in your handler, simply use it:
```
context.connection.read_sql_query(...)
```